src/components/cluster/detail/view.js

```diff
@@ -193,7 +193,10 @@ class ClusterDetailView extends React.Component {
       var workers = 0;
       nodes.forEach(node => {
         if (Object.keys(node).includes('labels')) {
-          if (node.labels['role'] != 'master') {
+          if (
+            node.labels['role'] != 'master' &&
+            node.labels['kubernetes.io/role'] != 'master'
+          ) {
```

Comment:
Why do you need to check both labels? I dont see same logic in gsctl 🤔

Reply:
I added effectively the same logic in gsctl here:
https://github.com/giantswarm/gsctl/pull/352/files

The reason is that label name changed with k8s 1.14.

Reply:
So basically it can be either `role` or `kubernetes.io/role` depending on the cluster version right?

Reply:
Yes, exactly.